spec/spec_helper.rb

```diff
@@ -28,7 +28,7 @@
   #   - http://rspec.info/blog/2014/05/notable-changes-in-rspec-3/#zero-monkey-patching-mode
   config.disable_monkey_patching!
 
-  config.warnings = true
+  config.warnings = false
```

Comment:
I know I did this, but perhaps a comment here to say set to false because ruby/rdf gems give a lot of warning messages